### PR TITLE
Update IPAM and configuration example

### PR DESCRIPTION
**Pull request by philsbln**
Addresses some Feedback @marwinski and @mandelsoft

### CODE REVIEW

File: 18-dualstack_ipv4_ipv6_seed_shoot.md [hunk 1]
```diff
@@ -27,7 +27,23 @@ The easiest solution is to provide both protocol stacks - IPv4/IPv6 dual stack.
 
 ### Configuration
 
-> TODO
+In the shoot specification we will use the definition of the `network` section to provide the cidrs for `nodes`, `pods` and `services`. As those fields already exist they will be enhanced to accept a tuple of values. One value for IPv4 and one for IPv6. We introduce special key words to request different strategies for choosing the respective CIDRs. These will be available at runtime only and being configured by Gardener/K8S during the cluster deployment/reconciliation.
+
+The following keywords are currently anticipated
+- **`autoV4`/`autoV6`** use the cloud provider's IPAM to reserve a prefix for the range (optional: of at least that size).
```
Comment (marwinski) [inline 18-dualstack_ipv4_ipv6_seed_shoot.md]: autoV4 will always be private ranges while autoV6 will probably be a public range. This is quite a difference. when using IPv4 addresses will not be managed or assigned by the cloud provider

File: 18-dualstack_ipv4_ipv6_seed_shoot.md [hunk 2]
```diff
@@ -27,7 +27,23 @@ The easiest solution is to provide both protocol stacks - IPv4/IPv6 dual stack.
...
+- **`noV4`/`noV6`** disable IPv4/IPv6 support
```
Comment (marwinski) [inline 18-dualstack_ipv4_ipv6_seed_shoot.md]: shouldn't we only specify what we enable? This doubles it.

File: 18-dualstack_ipv4_ipv6_seed_shoot.md [hunk 3]
```diff
@@ -27,7 +27,23 @@ The easiest solution is to provide both protocol stacks - IPv4/IPv6 dual stack.
...
+- **`+nodePrefix`** modifyer – request a prefix per node (optional: of at least that size)
```
Comment (marwinski) [inline 18-dualstack_ipv4_ipv6_seed_shoot.md]: Well, isn't this dependent on the cloud provider? For IPv4 this also depends on the CNI provider.

Comment (philsbln) [reply 18-dualstack_ipv4_ipv6_seed_shoot.md]: It will be… but at least for AWS, there is some flexibility (between /64 and /80)

**Comment by philsbln** [comment]
As we are in an early phase, I am merging this anyway and wait for discussions later

**Pull request merged**